package.json

```diff
@@ -19,13 +19,16 @@
     "docs:typescript:check": "yarn workspace docs typescript",
     "docs:typescript:formatted": "yarn workspace docs typescript:transpile",
     "docs:mdicons:synonyms": "babel-node --config-file ./babel.config.js ./docs/scripts/updateIconSynonyms",
+    "framer": "yarn framer:core && yarn prettier:framer",
+    "framer:core": "cross-env BABEL_ENV=docs-development babel-node ./framer/scripts/buildFramer.js ./packages/material-ui/src ./framer/Material-UI.framerfx/code",
```

Comment:
I've moved them to the framer workspace, but I can't seem to make it work (I was originally following `docs:api` which is in the root workspace).

```json
# framer/package.json
{
  "[...]": {"[...]"},
  "devDependencies": {
    "case": "1.6.2",
    "docs": "^4.0.0"
  },
  "scripts": {
    "build": "yarn build:core && yarn prettier",
    "build:core": "cross-env BABEL_ENV=development babel-node --config-file ../babel.config.js ./scripts/buildFramer.js ../packages/material-ui/src ./Material-UI.framerfx/code",
    "prettier": "prettier --write --config ../prettier.config.js 'framer/**/*.{js,tsx}'"
  }
}
```

```sh
> yarn build                                                                        

yarn run v1.19.0
$ yarn build:core && yarn prettier
$ cross-env BABEL_ENV=development babel-node --config-file ../babel.config.js ./scripts/buildFramer.js ../packages/material-ui/src ./Material-UI.framerfx/code
/Users/matthew.brookes/Projects/material-ui/docs/src/modules/utils/defaultPropsHandler.js:1
import * as astTypes from 'ast-types';
       ^

SyntaxError: Unexpected token *
```

Any hints? (Otherwise I can push in a broken state and you can take a look.)